compiler/codegen/symbol.cpp

```diff
@@ -794,7 +812,9 @@ GenRet ArgSymbol::codegenType() {
   Type* useType = getArgSymbolCodegenType(this);
 
   if( outfile ) {
-    ret.c = useType->codegen().c;
+    std::string argType = transformTypeForPointer(useType,
+                                                  this->defPoint->parentSymbol->hasFlag(FLAG_EXPORT));
```

Comment:
What happens if you just pass 'true' here. Does testing pass? I don't think this behavior should depend on FLAG_EXPORT.

Reply:
I haven't tried that (wanted to make sure it worked in the export case before expanding), but I will now.

Reply:
Full local paratest passed.  I'm going to remove the argument from the function